drake/util/drakeGeometryUtil.h

```diff
@@ -619,19 +620,19 @@ void angularvel2rpydotMatrix(const Eigen::MatrixBase<DerivedRPY>& rpy,
   Scalar cp = cos(p);
   Scalar tp = sp / cp;
 
-  phi << cy / cp, sy / cp, 0.0, -sy, cy, 0.0, cy * tp, tp * sy, 1.0;
+  phi << cy / cp, sy / cp, Scalar(0), -sy, cy, Scalar(0), cy * tp, tp * sy, Scalar(1);
```

Comment:
The Scalar() thing was necessary to fix an issue with second order AutoDiffScalar derivatives.